src/imitation/rewards/serialize.py

```diff
@@ -174,13 +174,13 @@ def load_zero(path: str, venv: VecEnv) -> reward_function.RewardFn:
     del path, venv
 
     def f(
-        obs: np.ndarray,
-        act: np.ndarray,
-        next_obs: np.ndarray,
-        dones: np.ndarray,
+        state: np.ndarray,
```

Comment:
Maybe this is worth making positional only in the original type and everywhere else?

Reply:
I'm always a bit scared of positional functions where all arguments have the same type, very easy to get them mixed up. I'd rather just actually stick to a convention here, the current one seems OK.